doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -29,6 +28,13 @@ box.info.synchro
             With elections enabled, an instance runs ``box.ctl.promote()`` command automatically after winning the elections.
             To clear the ownership, call :ref:`box.ctl.demote() <box_ctl-demote>` on the synchronous queue owner.
 
+            ..  note::
+
+                When Raft election is enabled and :ref:`replication.election_mode <configuration_reference_replication_election_mode>`
+                is set to ``candidate``, the new Raft leader claims the queue automatically.
```

Comment:
I think this paragraph reiterates the one above (starting with `With elections enabled`).

Let's merge them. Also I like your explanation more than the one above.

Reply:
Thanks, fixed